qualang_tools/bakery/bakery.py

```diff
@@ -13,22 +13,20 @@ def baking(
     config,
     padding_method="right",
     override=False,
-    update_config: bool = True,
     baking_index: int = None,
 ):
     """
-    Opens a context manager to synthesize samples for arbitrary waveforms
+    Opens a context manager to synthesize samples for arbitrary waveforms. No return, config is updated if update_config is set to True,
```

Comment:
There is no update_config now :)
You can write:
Opens a context manager to synthesize samples for arbitrary waveforms. The inputted config is updated, unless a a baking_index is given, in which case the generated waveforms can be retrieved using the get_waveform_dict() method, in a format readily plugable as an overrides argument